easybuild/easyconfigs/p/pytorch-3dunet/pytorch-3dunet-1.6.0-foss-2022a-CUDA-11.7.0.eb

```diff
@@ -0,0 +1,57 @@
+# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2023/11
+easyblock = 'PythonBundle'
+
+name = 'pytorch-3dunet'
+version = '1.6.0'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://github.com/wolny/pytorch-3dunet'
+description = """
+PyTorch implementation of 3D U-Net and its variants:
+ - UNet3D: Standard 3D U-Net based on 3D U-Net: Learning Dense Volumetric          Segmentation from Sparse Annotation 
+ - ResidualUNet3D: Residual 3D U-Net based on Superhuman Accuracy on the SNEMI3D
+   Connectomics Challenge 
+ - ResidualUNetSE3D: Similar to ResidualUNet3D with the addition of Squeeze and 
+   Excitation blocks based on Deep Learning Semantic Segmentation for High-     
+   Resolution Medical Volumes. Original squeeze and excite paper: Squeeze-and-  
+   Excitation Networks
+The code allows for training the U-Net for both: semantic segmentation (binary  
+and multi-class) and regression problems (e.g. de-noising, learning             
+deconvolutions).
+"""
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+builddependencies = [
+    ('pkgconf', '1.8.0'),
+    ('cppy', '1.2.1')
+]
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('PyTorch', '1.12.0', versionsuffix),
+    ('CUDA', '11.7.0', '', SYSTEM),
+    ('h5py', '3.7.0'),
+    ('tensorboard', '2.10.0'),
+    ('scikit-image', '0.19.3'),
+    ('SciPy-bundle', '2022.05'),
```

Comment:
Put this right after Python, a couple of the above deps depend on it so to reduce module reloading it should be higher up.